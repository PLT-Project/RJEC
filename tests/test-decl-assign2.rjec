func main() {

    var x, y int = 42, 30;
    printi(x);
    printi(y);

    var r, j, e, c char = 'r', 'j', 'e', 'c';
    printc(r);
    printc(j);
    printc(e);
    printc(c);

}