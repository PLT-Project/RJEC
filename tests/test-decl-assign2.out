42
30
r
j
e
c